integration-tests/src/test/java/io/quarkus/qe/HealthIT.java

```diff
@@ -8,17 +8,9 @@
     private static final String HEALTH_PATH = "/health";
 
     @Test
-    public void storageServiceHealthEndpointShouldBeOk() {
+    public void healthEndpointsShouldBeOk() {
         awaitFor(() -> givenStorageService().get(HEALTH_PATH).then().statusCode(HttpStatus.SC_OK));
-    }
-
-    @Test
-    public void restApiHealthEndpointShouldBeOk() {
         awaitFor(() -> givenRestApiService().get(HEALTH_PATH).then().statusCode(HttpStatus.SC_OK));
-    }
-
-    @Test
-    public void enricherServiceHealthEndpointShouldBeOk() {
```

Comment:
This change is because now we're going to start and stop the whole deployment for each test, so doing these checks in a single test is more efficient.